modules/va_facilities/README.v1.yml

```diff
@@ -86,27 +88,37 @@ paths:
       security:
         - api_key: []
       parameters:
-        - name: lat
+        - name: address_1
```

Comment:
lacking an address_2, address_1 seems out of place .. address or street_address seem to be common on my quick search of address fields

Reply:
I was hoping someone would question this! I only went with address_1 because that's what it is called in the Facilities schema, but I agree if feels icky without additional address fields. I much prefer street_address (address feels a little too vague to me).